app/views/collaborators/_add.html.erb

```diff
@@ -3,10 +3,15 @@
 
 <%= form_for Collaborator.new do |f| %>
   <div class="row collapse">
-    <div class="small-10 columns">
+    <div class="small-12 columns">
       <%= f.hidden_field :resourceable_type, value: resource.class.name %>
       <%= f.hidden_field :resourceable_id, value: resource.id %>
-      <%= f.hidden_field :user_ids, class: 'collaborators multiple', 'data-url' => collaborators_path(ineligible_user_ids: Collaborator.ineligible_collaborators_for(resource).map(&:id)) %>
+      <%= f.label :users %>
+      <%= f.hidden_field :user_ids, id: 'user_ids', class: 'collaborators multiple', 'data-url' => collaborators_path(ineligible_user_ids: Collaborator.ineligible_collaborators_for(resource).map(&:id)) %>
+      <% if ROLLOUT.active?(:collaborator_groups) %>
```

Comment:
Adds a groups field to the add a collaborator form.  Only shows the field if the collaborators_groups feature is active.